src/test/a11y/a11y.ts

```diff
@@ -92,7 +93,8 @@ const excludedPaths: DefendantResponsePaths[] = [
   DefendantResponsePaths.receiptReceiver,
   DefendantResponsePaths.legacyDashboardRedirect,
   OfferPaths.agreementReceiver,
-  DefendantFirstContactPaths.receiptReceiver
+  DefendantFirstContactPaths.receiptReceiver,
+  ClaimantResponsePaths.checkAndSendPage
```

Comment:
Why check-and-send is excluded?

Reply:
not implemented yet.

Reply:
We just need to remember to revert it when it is implemented so we are checking it as a part of a11y tests.

Reply:
can we not exclude it please? that's something easily forgotten

Reply:
don't worry, it won't be forgotten, at this point it would be un necessary work-around to to include and make it pass.